lib/pgBackRest/Config/Config.pm

```diff
@@ -2121,9 +135,9 @@ sub configLogging
 push @EXPORT, qw(configLogging);
 
```

Comment:
Not sure where to enter this issue and I haven't had a chance to chase it down myself, but if I mistype the db-path as db-path1 instead of db1-path, I get the following error in addition to "Unknown option":

postgres@db-master:~$ pgbackrest --db-path1=/var/lib/postgresql/9.5/demo --stanza=demo --log-level-console=info check
Unknown option: db-path1
ERROR [124]: : process terminated due to an unhandled exception:
Use of uninitialized value $strCommand in string eq at /usr/share/perl5/pgBackRest/Config/Config.pm line 1139.
 at /usr/bin/pgbackrest line 15.
	main::__ANON__("Use of uninitialized value \$strCommand in string eq at /usr/s"...) called at /usr/share/perl5/pgBackRest/Config/Config.pm line 1139
	pgBackRest::Config::Config::cfgCommandTest(5) called at /usr/bin/pgbackrest line 40
	eval {...} called at /usr/bin/pgbackrest line 282

It works fine if I type
 pgbackrest --db1-path=/var/lib/postgresql/9.5/demo --stanza=demo --log-level-console=info check